src/components/molecules/Visualizer/Engine/Cesium/Tileset/index.tsx

```diff
@@ -13,25 +26,109 @@ export type Property = {
   const { isVisible, property } = layer ?? {};
-  const { sourceType, tileset, styleUrl, shadows } =
+  const { sourceType, tileset, styleUrl, shadows, edgeColor, edgeWidth, clippingPlaneCollection } =
     (property as Property | undefined)?.default ?? {};
+  const {
+    dimensions: planeDimensions,
+    lat,
+    lng,
+    height,
+    heading,
+    roll,
+    pitch,
+    planes: _planes,
+  } = clippingPlaneCollection || {};
   const [style, setStyle] = useState<Cesium3DTileStyle>();
+  const prevPlanes = useRef(_planes);
+  const planes = useMemo(() => {
+    if (
+      !prevPlanes.current?.length ||
+      !prevPlanes.current?.every(
+        (p, i) =>
+          p.normal?.x === _planes?.[i].normal?.x &&
+          p.normal?.y === _planes?.[i].normal?.y &&
+          p.normal?.z === _planes?.[i].normal?.z &&
+          p.distance === _planes?.[i].distance,
+      )
+    ) {
+      prevPlanes.current = _planes;
+    }
+    return prevPlanes.current;
+  }, [_planes]);
+  // Create immutable object
+  const [clippingPlanes] = useState(
+    () =>
+      new CesiumClippingPlaneCollection({
+        planes: planes?.map(
+          plane =>
+            new ClippingPlane(
+              new Cartesian3(plane.normal?.x, plane.normal?.y, plane.normal?.z),
+              (plane.distance || 0) * -1,
+            ),
+        ),
+        edgeWidth: edgeWidth,
+        edgeColor: toColor(edgeColor),
+      }),
+  );
+  const tilesetRef = useRef<Cesium3DTilesetType>();
 
   const ref = useCallback(
     (tileset: CesiumComponentRef<Cesium3DTilesetType> | null) => {
       if (layer?.id && tileset?.cesiumElement) {
         (tileset?.cesiumElement as any)[layerIdField] = layer.id;
       }
+      tilesetRef.current = tileset?.cesiumElement;
     },
     [layer?.id],
   );
 
+  useEffect(() => {
+    const prepareClippingPlanes = async () => {
+      if (!tilesetRef.current) {
+        return;
+      }
+
+      await tilesetRef.current?.readyPromise;
+
+      const clippingPlanesOriginMatrix = Transforms.eastNorthUpToFixedFrame(
+        tilesetRef.current.boundingSphere.center.clone(),
+      );
+
+      const dimensions = new Cartesian3(
+        planeDimensions?.width || 100,
+        planeDimensions?.length || 100,
+        planeDimensions?.height || 100,
+      );
+
+      const position = Cartesian3.fromDegrees(lng || 0, lat || 0, height);
+
+      const hpr =
+        heading && pitch && roll ? HeadingPitchRoll.fromDegrees(heading, pitch, roll) : undefined;
+      const boxTransform = Matrix4.multiply(
+        hpr
+          ? Matrix4.fromRotationTranslation(Matrix3.fromHeadingPitchRoll(hpr), position)
+          : Transforms.eastNorthUpToFixedFrame(position),
+        Matrix4.fromScale(dimensions, new Matrix4()),
+        new Matrix4(),
+      );
+
+      const inverseOriginalModelMatrix = Matrix4.inverse(clippingPlanesOriginMatrix, new Matrix4());
+
+      Matrix4.multiply(inverseOriginalModelMatrix, boxTransform, clippingPlanes.modelMatrix);
```

Comment:
This is also the same with the above comment.
The reference for `clippingPlanes.modelMatrix` get updated.